tests/queries/test_bulk_update.py

```diff
@@ -300,6 +300,33 @@ def test_json_field(self):
             JSONFieldNullable.objects.filter(json_field__has_key="c"), objs
         )
 
+    @skipUnlessDBFeature("supports_json_field")
+    def test_json_field_sql_null_value(self):
+        """
+        Setting a JSONField to None results in SQL NULL.
+        """
+        obj = JSONFieldNullable.objects.create(json_field={})
+        obj.json_field = None
+        JSONFieldNullable.objects.bulk_update([obj], fields=["json_field"])
+        obj.refresh_from_db()
+        sql_null_qs = JSONFieldNullable.objects.filter(json_field__isnull=True)
+        self.assertSequenceEqual(sql_null_qs, [obj])
+
+    @skipUnlessDBFeature("supports_json_field")
+    def test_json_field_json_null_value(self):
+        """
+        Setting a JSONField to Value(None, output_field=JSONField()) results
+        in a JSON null.
+        """
+        obj = JSONFieldNullable.objects.create(json_field={})
+        obj.json_field = Value(None, output_field=JSONField())
+        JSONFieldNullable.objects.bulk_update([obj], fields=["json_field"])
+        obj.refresh_from_db()
+        json_null_qs = JSONFieldNullable.objects.filter(
+            json_field=Value(None, output_field=JSONField())
```

Comment:
I _think_ this should work with

```suggestion
            json_field=None
```

as we differentiate `None` handling between saving and querying (yeah it's weird)